animal-info.js

```diff
@@ -0,0 +1,104 @@
+const animals = {
+  reptiles: {
+    frillNeckedLizard: {
+      lifespan: "20 years",
+      group: "reptile",
+      food: ["Small insects", "spiders"],
+      description:
+        "When this unique creature feels threatened, it rises on its hind legs, opens its yellow-coloured mouth, unfurls the colorful, pleated skin flap that encircles its head, and hisses. If an attacker is unintimidated by these antics, the lizard simply turns tail, mouth and frill open, and bolts, legs splaying left and right. It continues its deliberate run without stopping or looking back until it reaches the safety of a tree.",
+      length: "90cm",
+      weight: "1kg",
+      found: "Northern Australia",
+    },
+    hawksbillTurtle: {
+      lifespan: "50 years",
+      group: "reptile",
+      food: ["sponges", "jellyfish", "sea plants"],
+      description:
+        "The Hawksbill Sea Turtle gets its common name from the distinctive mouth, which resembles a bird’s beak. The shell, or carapace, is covered in large overlapping scutes, which are a distinctive brown/green/amber background with lighter brown streaks throughout. At the edge of the carapace they overlap in such a way as to form a serrated edge. The turtle has an elongated head and flippers which have two visible claws on the end. The flippers and head are covered in large green, brown or yellow scales. The average adult female weighs 50 kg and their carapace (shell) is approximately 80 cm in length.",
+      length: "80cm (carapace)",
+      weight: "50kg",
+      found:
+        "Tropical coasts of Queensland, Northern Territory and Western Australia",
+    },
+    perentie: {
+      lifespan: "20 years",
+      group: "reptile",
+      food: ["kangaroos", "rabbits", "lizards", "birds"],
+      description:
+        "The perentie (Varanus giganteus) is the largest monitor lizard or goanna native to Australia. It is one of the largest living lizards on earth, after the Komodo dragon, Asian water monitor, crocodile monitor, and intersecting by size with Nile monitor. Found west of the Great Dividing Range in the arid areas of Australia, it is rarely seen, because of its shyness and the remoteness of much of its range from human habitation. The species is considered to be a least-concern species according to the International Union for Conservation of Nature. Its status in many Aboriginal cultures is evident in the totemic relationships, and part of the Ngiṉṯaka dreaming, as well as bush tucker. It was a favoured food item among desert Aboriginal tribes, and the fat was used for medicinal and ceremonial purposes.",
+      length: "2.5m",
+      weight: "20kg",
+      found: "Deserts",
+    },
+  },
+  mammals: {
+    echidna: {
```

Comment:
You have an extra open bracket without a corresponding closing bracket. This mismatch will cause a syntax error and prevent the code from running correctly when we try to use this list of objects, remove and resubmit. Line 36.